example/etcd-statefulset-aws.yaml

```diff
@@ -23,23 +23,102 @@ metadata:
 data:
   bootstrap.sh: |-
```

Comment:
It is generated from the template file. So the code generator generates across all the cloud providers. This will be changed when the helm charts are introduced.